### PR TITLE
Unable to stop the kolibri server

### LINKED ISSUE #7: Unable to stop the server: kolibri-stop.bat in the wrong Python folder
Issue from: [Unable to stop the server: kolibri-stop.bat in the wrong Python folder #1222](https://github.com/learningequality/kolibri/issues/1222)

## Summary

My guess is that this is because there was the switch from Python 3 to Python 2, and the `kolibri-stop.bat` script was left behind. 

Tried and failed to stop the server form the task tray. Opened the `Python27/Scripts` folder to stop it from the command prompt and noticed there was a `Python34/Scripts` folder even though not Python 3 was installed on the system. Run the `kolibri-stop.bat` script thatwas inside and the server stopped.

## System information

 - Version: `0.4.0alpha1.dev511`
 - Operating system: Windows 7
 - Browser: all

## Screenshots

virtualbox_ie11 - win7_13_04_2017_17_35_12

/cc @radinamatic

### PR DESCRIPTION
**Pull request by mrpau-richard**
Fix kolibri stop script path destination.

Fixes #7 

/cc @cpauya

### CODE REVIEW

**Comment by cpauya** [comment]
@mrpau-richard I searched for possible occurrences of `python*` in the source based on `develop` branch and found at https://github.com/learningequality/kolibri-installers/blob/4083bca34412179661443a1f23f0b8cb8e3c4fad/windows/gui-source/Kolibri/Kolibri.cpp#L44

> ...
> 	char script_dirs[26] = "c:\\Python34\\Scripts";
> ...

Since we downgraded to at least v2.7.10, can you double-check that and change it too?

**Comment by mrpau-richard** [comment]
@cpauya I've updated the Kolibri windows application and source.

File: windows/gui-source/Kolibri/Kolibri.cpp [hunk 1]
```diff
@@ -41,7 +41,7 @@ void kolibriScriptPath(char *buffer, const DWORD MAX_SIZE)
 		sprintf(err_message, "Error: the value of KOLIBRI_SCRIPT_DIR must be less than %d, but it was length %d. Please start Kolibri from the command line.", MAX_SIZE, bufsize);
 		window->sendTrayMessage("Kolibri", err_message);
 	}
-	char script_dirs[26] = "c:\\Python34\\Scripts";
```
Comment (cpauya) [inline windows/gui-source/Kolibri/Kolibri.cpp]: Can we find a way to remove the hard-coded `c:\` drive?  What if the user installed `Python` somewhere?

You can file a separate issue for this after testing, so we can close this.

Comment (mrpau-richard) [inline windows/gui-source/Kolibri/Kolibri.cpp]: I think I will create another issue for this and fix it there.

**Comment by mrpau-richard** [comment]
@radinamatic can you test the fix on this [issue](https://github.com/learningequality/kolibri-installers/issues/7) using this [kolibri windows installer](https://drive.google.com/open?id=0B5Tyi-tRKOFMeHpFWjBlNGRlWXM).

**Comment by radinamatic** [comment]
Fix confirmed! No sign of the old `Python34` folder, `kolibri-stop.bat` where it is supposed to be! 👍 

cursor_y_windows_7__start___running_

**Comment by mrpau-richard** [comment]
@cpauya I think this PR is ready to be merge.

**Pull request merged**